server/src/weaverbird/backends/mongo_translator/steps/rank.py

```diff
@@ -0,0 +1,91 @@
+from typing import List
+
+from weaverbird.backends.mongo_translator.steps.types import MongoStep
+from weaverbird.pipeline.steps import RankStep
+
+
+def translate_rank(step: RankStep) -> List[MongoStep]:
+    vqb_var_order: MongoStep = {}
+
+    if step.method == 'dense':
+        vqb_var_order = {
+            '$cond': [
+                {'$ne': [f'$$this.{step.value_col}', '$$value.prevValue']},
```

Comment:
doesn't the double `$$` in a JS backtick string escape to a single `$` ?

Reply:
https://github.com/ToucanToco/weaverbird/pull/1310#discussion_r863061239